test/util/cli_helpers.py

```diff
@@ -41,6 +41,7 @@ def test_wrapper_func(self, *inner_args, **inner_kwargs):
             client_instance_mock = MagicMock()
             client_instance_mock.get_many.return_value = get_many_output
             client_instance_mock.list.return_value = list_output
+            client_instance_mock.whoami.return_value = whoami_output
```

Comment:
Move this to the bottom of the list since it's defined last in api/client files (so they're easier to match up)